packages/cli/src/test-utils/chain-setup.ts

```diff
@@ -116,3 +125,24 @@ export const voteForGroupFromAndActivateVotes = async (
 export const mineEpoch = async (kit: ContractKit) => {
   await mineBlocks(100, kit.web3)
 }
+
+export const topUpWithToken = async (
+  kit: ContractKit,
+  stableToken: StableToken,
+  account: string,
+  amount: BigNumber
+) => {
+  const token = await kit.contracts.getStableToken(stableToken)
+
+  await impersonateAccount(kit.web3, STABLES_ADDRESS)
+  await token.transfer(account, amount.toFixed()).sendAndWaitForReceipt({
+    from: STABLES_ADDRESS,
+  })
+  await stopImpersonatingAccount(kit.web3, STABLES_ADDRESS)
+}
+
+export const setupL2 = async (kit: ContractKit) => {
+  // Temporarily deploying any bytecode, so it's just there,
```

Comment:
eventually i imagine the anvil devchain will come preloaded with something deployed here. at that time we can remove the setupL2 function i imagine.

Reply:
I'll add a TODO